modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureListView.testcase

```diff
@@ -34,188 +38,172 @@ definition {
+	@description = "LPS-131206. Verify if Digital Signature can be enabled"
 	@priority = 5
 	test DigitalSignatureCanBeEnabled {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
+		task ("When a User Test Goes to Digital Signature") {
+			Navigator.openURL();
 
-		Navigator.openURL();
-
-		ProductMenu.gotoPortlet(
-			category = "Content & Data",
-			portlet = "Digital Signature");
-
-		var dsStatus = "Sent";
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Digital Signature");
+		}
 
-		AssertElementPresent(
-			dsStatus = ${dsStatus},
-			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_STATUS");
+		task ("Then the status is checked") {
+			DigitalSignature.viewDigitalSignaturestatus(viewStatus = "Sent");
+		}
 	}
 
-	@description = "This is a test for LPS-137169. Verify if have more recipients than one"
+	@description = "LPS-131206. Verify if have more recipients than one"
 	@priority = 4
 	test HaveMoreRecipientsThanOne {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
-
-		DigitalSignature.createEnvelope(
-			addEmail = "test@liferay.com",
-			addFullName = "Recipient Full Name",
-			addMessage = "Email Message",
-			addName = "Poshi Test HaveMoreRecipientsThanOne",
-			addSubject = "Email Subject",
-			dmDocumentFile = "Alice's Adventures in Wonderland.pdf");
-
-		Click(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_ADD_RECIPIENT");
-
-		Type(
-			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_EXTRA_RECIPIENT_NAME",
-			value1 = "Recipient Full Name 2");
-
-		Type(
-			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_EXTRA_RECIPIENT_EMAIL",
-			value1 = "test2@liferay.com");
-
-		Button.click(button = "Send");
-
-		Alert.viewSuccessMessageText(successMessage = "Your envelope was created successfully.");
-
-		Refresh();
+		task ("Given a digital envelope is created") {
+			DigitalSignature.createEnvelope(
+				addEmail = "test@liferay.com",
+				addFullName = "Recipient Full Name",
+				addMessage = "Email Message",
+				addName = "Poshi Test HaveMoreRecipientsThanOne",
+				addSubject = "Email Subject",
+				dmDocumentFile = "Alice's Adventures in Wonderland.pdf");
+		}
 
-		WaitForElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_RECIPIENTS_BADGE");
+		task ("And add another recipient") {
+			DigitalSignature.addMoreRecipient(
+				fullName = "Recipient Full Name 2",
+				recipientEmail = "test2@liferay.com");
+		}
 
-		AssertElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_RECIPIENTS_BADGE");
+		task ("When a message appears that the envelope was successfully created") {
+			Alert.viewSuccessMessageText(successMessage = "Your envelope was created successfully.");
+		}
 
-		DigitalSignature.deleteEnvelope(envelopeName = "Poshi Test HaveMoreRecipientsThanOne");
+		task ("Then it shows that more than one recipient has been created") {
+			Refresh();
```

Comment:
Could you change this task to: `Then the user can see that  an envelope have more than one recipient on list`